draft-ietf-quic-manageability.md

```diff
@@ -557,19 +564,20 @@ future versions of the protocol.
 
 ## Connection confirmation {#sec-confirm}
 
-Connection establishment uses Initial, Handshake, and Retry packets containing a
+Connection establishment uses Initial and Handshake packets containing a
```

Comment:
It really does include Retry.  Though those don't include TLS handshake messages.